tools/bumper/bumper.go

```diff
@@ -83,11 +83,14 @@ func main() {
 
 		if bumpNeeded {
 			logger.Printf("Bumping %s from %s to %s", componentName, currentReleaseTag, updatedReleaseTag)
-			// reset --hard git repo
-			exitWithError(fmt.Errorf("reset --hader repo not implemented yet"))
 
 			// Create PR
 			exitWithError(fmt.Errorf("create PR not implemented yet"))
+			err = bumpingRepo.resetRepo()
```

Comment:
Why is this not before the `// Create PR` part?

Reply:
it shouldn't be.
Done